pyproject.toml

```diff
@@ -66,6 +57,18 @@ sodapy = "^2.2.0"
 pangres = "^4.1.2"
 ngboost = "^0.3.13"
 ibis-framework = "^3.2.0"
+pytrends = {version = "^4.7.3", optional = true}
+jupyter = {version = "^1.0.0", optional = true}
+GeoAlchemy2 = {version = "^0.10.0", optional = true}
+matplotlib = {version = "^3.5.1", optional = true}
+plotly =  {version = "^5", optional = true}
+geopandas =  {version = "^0.10.2", optional = true}
+tensorflow = {version = "^2.5.0", optional = true}
+scikit-learn = {version = "^1.0.2", optional = true}
+
+[tool.poetry.extras]
+ml = ["pytrends", "jupyter", "GeoAlchemy2", "matplotlib", "plotly", "geopandas", "tensorflow",
```

Comment:
@fccoelho let me know if there are any dependencies you think that shouldn't be here (or those who should)

Reply:
ngboost could also be added to the ML extras, matplotlib, plotly and arviz could be on a `viz` Extra package, instead of the `ml`,

Geoalchemy, jupyter and geopandas are not  `ml` packages either.